.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+Before submitting a PR make sure the following things have been done (and denote this
+by checking the relevant checkboxes):
+
+- [ ] The commits are consistent with our [contribution guidelines](CONTRIBUTING.md)
+- [ ] You've added tests (if possible) to cover your change(s)
+- [ ] The new code is not generating byte compile warnings (run `cask exec emacs -batch -Q -L . -eval "(progn (setq byte-compile-error-on-warn t) (batch-byte-compile))" clj-refactor.el`)
+- [ ] The new code is not generating `M-x checkdoc` warnings
```

Comment:
We don't have a checkdoc config and I find the defaults slightly retarded, so I'd prefer we remove this.